src/Plugin/RulesAction/DataListItemAdd.php

```diff
@@ -44,13 +44,20 @@
 class DataListItemAdd extends RulesActionBase {
 
   /**
-   * {@inheritdoc}
+   * Executes the Plugin.
+   *
+   * @param array $list
+   *   An array to add an item to.
+   * @param mixed $item
+   *   An item which is being added to the array.
+   * @param bool $unique
+   *   A boolean value which indicates if the item should be updated if it is
+   *   already in the list array.
+   * @param string $pos
+   *   Position in array where the item is being added.
```

Comment:
This should document the available options as well that it it's optional. Also is unique not optional also?